app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,8 @@
+<div><strong>post #<%=post.id%></strong><br><%=post.text...%><br>Comments: <%=post.comments_counter%>, Likes: <%=post.likes_counter%>
+<ol>
+<%post.comments.each do |comment|%>
+<li><%=comment.author.name%>: <%=comment.text%></li><br>
+<%end%>
+</ol>
+</div><br>
+
```

Comment:
- [ ] Please fix the following error that appears when you first migrate the database.
![image](https://user-images.githubusercontent.com/62290545/199198306-9ac5f07d-b6d3-4b3f-91c5-d189ca26ba3a.png)


- To replicate the error, first, you need to drop the DB, recreate it, and then migrate again and you'll see the error in the console.

- [HINT] This error is caused by the order of the migrations you have. As you can see below, the order of migrations is missed up because you remove the `name` column before adding it to the `users` table. I don't know why you have the removal migration but if you delete this migration it should fix the error.


![image](https://user-images.githubusercontent.com/62290545/199199065-e9ac8d57-0471-49cb-8318-25ab1148477c.png)